clang/test/C/C2y/n3369.c

```diff
@@ -102,6 +126,11 @@ void test_unspecified_array_length() {
   static_assert(_Countof(**x) == 3);
 }
 
+void test_completed_array() {
+  int a[] = {1, 2, errno};
```

Comment:
Best not to use `errno` here, just replace with `3`.

Reply:
Ah, in that case, go with a local or global variable (we don't typically pull in standard library headers as part of our tests because that ends up relying on whatever is installed on the system, which makes the tests non-hermetic).